src/sentry/api/endpoints/group_details.py

```diff
@@ -272,7 +272,7 @@ def put(self, request, group):
         :param string status: the new status for the issue.  Valid values
                               are ``"resolved"``, ``resolvedInNextRelease``,
                               ``"unresolved"``, and ``"ignored"``.
-        :param string assignedTo: the username of the user that should be
+        :param string assignedTo: the id of the user that should be
```

Comment:
Isn't this incorrect? Won't it be related to team or user, and using the new syntax `user:1` vs `team:1 ?

Reply:
I guess I'd say just copy/paste from the project_group_index docstring now.